### PR TITLE
Update custom mouse buttons article

### PR DESCRIPTION
**Pull request by ahoneybun**
This PR will add some GUI (graphical user interface) tools to adjust the buttons on *some* mice like the MX Master 3 that we sell on our website.

### CODE REVIEW

File: _articles/custom-mouse-buttons.md [hunk 1]
```diff
@@ -1,20 +1,51 @@
 ---
 layout: article
-title: Setting Custom Mouse Actions
+title: Customizing Your Mouse
 description: >
-   How to set custom mouse actions with xbindkeys
+   How to custom the buttons on your mouse
```
Comment (jacobgkau) [inline _articles/custom-mouse-buttons.md:5]: `custom` is not a verb, this should probably say `How to customize`.

Comment (ahoneybun) [reply _articles/custom-mouse-buttons.md]: Should be fixed now.

File: _articles/custom-mouse-buttons.md [hunk 2]
```diff
@@ -1,20 +1,51 @@
...
 ---
 
-### Software
+### Graphical Software
+
+#### Piper
+
+![Piper Buttons Window](/images/custom-mouse-buttons/Piper-2.png) 
+
+<u>Piper</u> is a frontend to the <u>libratbag</u> backend for setting custom actions to mouse buttons like the Logitech MX Master 3 mouse that we sell and this section will be using that mouse as an example for using <u>Piper</u>. You can see the source code of <u>Piper</u> [here](https://github.com/libratbag/piper).
```
Comment (storyaddict) [inline _articles/custom-mouse-buttons.md:25]: like the Logitech MX Master 3 mouse that we sell

Change to "like the Logitech MX Master 3 (sold by System76). In this section we will be...

File: _articles/custom-mouse-buttons.md [hunk 3]
```diff
@@ -1,20 +1,51 @@
...
+
+In this window the button that has been clicked can be reassigned to a differet function like changing the Left-click to the Right-click or swapping the Forward button to the Back button for web pages.
+
+#### Solaar
+
+![Solaar Main Window](/images/custom-mouse-buttons/Solaar.png) 
+
+![Solaar Main Window with information](/images/custom-mouse-buttons/Solaar-2.png) 
+
+<u>Solaar</u> is a graphical tool to for adding more devices to your Logitech receiver and information like the following:
```
Comment (storyaddict) [inline _articles/custom-mouse-buttons.md:37]: is a graphical tool to for adding more devices to your Logitech receiver and information like the following:

Change to:
is a graphical tool for adding more devices to your Logitech USB Unify receiver. It also provides information on the following:

**Review by storyaddict — CHANGES_REQUESTED** [review]

File: _articles/custom-mouse-buttons.md (hunk 2 above)
Comment (storyaddict) [reply _articles/custom-mouse-buttons.md:25]: This is still a run-on sentence.

File: _articles/custom-mouse-buttons.md (hunk 3 above)
Comment (storyaddict) [reply _articles/custom-mouse-buttons.md:37]: This is also still a run-on/incompatible sentence. 

right now it reads: adding more devices to your receiver and [adding more devices to your] information.

**Review by storyaddict — CHANGES_REQUESTED** [review]
Changes not implemented.

**Review by Socialhappiness — APPROVED** [review]
Good job!

**Pull request merged**